Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -278,16 +273,10 @@ private void TryLoadPointerIfNeeded()
                 // pointer currently registered with FocusManager, we use it.
 
                 Pointer = FocusManager.Instance.TryGetSinglePointer();
-
-                if (Pointer == null)
```

Comment:
Yeah, by adding SimpleSinglePointerSelector to InputManager, that fixed the issues.  There were a few scenes that didn't have a cursor, but still used InputManager, so for those scenes, I disabled SimpleSinglePointerSelector accordingly.